### ISSUE TITLE
GLES2/WebGL1 unsupported shadow cube

### ISSUE THREAD

**Issue by muiroc** [godotengine/godot#22959]
**Godot version:**
4c1a5d9 debug javascript build with a [modified index.js](https://gist.github.com/muiroc/818798d9f314a130e164c7cd7d8aa39e)

**OS/device including version:**
Windows 10 Firefox/Chrome with Angle

**Issue description:**
Error in the web console (x30 times):

GL ERROR :GL_INVALID_OPERATION : glTexImage2D: invalid target for depth/stencil textures

which could generated by (the loops math adds up):

https://github.com/godotengine/godot/blob/43c28681add010b5daccbcfde3e3065d7be7b52c/drivers/gles2/rasterizer_scene_gles2.cpp#L3065

**Notes:**
Most likely this results in no omni shadows, but the mesh geometry is missing so can't report it.
Still need to try with dual paraboloid shadow mode

**Steps to reproduce:**
At initialization

**Comment by HugeGameArtGD** [comment]
I get these errors and a black screen:
```
[Exception... "Component returned failure code: 0x805e0006 [nsIChannel.asyncOpen2]"  nsresult: "0x805e0006 (<unknown>)"  location: "JS frame :: resource:///modules/FaviconLoader.jsm :: load :: line 104"  data: no]
Content Security Policy: Die Einstellungen der Seite haben das Laden einer Ressource auf https://avatars2.githubusercontent.com/favicon.ico blockiert ("default-src").
uncaught exception: abort(83). Build with -s ASSERTIONS=1 for more info.
TypeError: Cc['@mozilla.org/updates/update-checker;1'] is undefined[Weitere Informationen] ASRouterTargeting.jsm:66:9
Error: WebGL warning: shaderSource: Backslash is not valid in WebGL 1. cube.js:7:339972
Error: WebGL warning: texImage2D: With format DEPTH_COMPONENT24, this function may only be called with target=TEXTURE_2D, data=null, and level=0.
cube.js:7:340673
Error: WebGL warning: checkFramebufferStatus: This operation requires zeroing texture data. This is slow. cube.js:7:326107
Error: WebGL: No further warnings will be reported for this WebGL context. (already reported 32 warnings) cube.js:7:326107
TypeError: JSEvents.lastGamepadState is undefined cube.js:7:204684
Error: WebGL warning: shaderSource: Backslash is not valid in WebGL 1. cube.js:7:339972
Error: WebGL warning: texImage2D: With format DEPTH_COMPONENT24, this function may only be called with target=TEXTURE_2D, data=null, and level=0.
cube.js:7:340673
Error: WebGL warning: checkFramebufferStatus: This operation requires zeroing texture data. This is slow. cube.js:7:326107
Error: WebGL: No further warnings will be reported for this WebGL context. (already reported 32 warnings) cube.js:7:326107
TypeError: JSEvents.lastGamepadState is undefined cube.js:7:204684
InvalidStateError: An attempt was made to use an object that is not, or is no longer, usable
Tastenereignis ist in manchen Tastaturlayouts nicht verfügbar: Taste="r" Modifikatoren="accel,alt" ID=“key_toggleReaderMode” browser.xul
Tastenereignis ist in manchen Tastaturlayouts nicht verfügbar: Taste="i" Modifikatoren="accel,alt,shift" ID=“key_browserToolbox” browser.xul
TypeError: Cc['@mozilla.org/updates/update-checker;1'] is undefined[Weitere Informationen]
```

**Comment by muiroc** [comment]
@HugeGameArtGD  yeah lately I also saw that weird Error: WebGL warning: shaderSource: Backslash is not valid in WebGL 1.
and chrome warned about some invalid character 0x22 in shader source, and in that shader source there was an include "somestuff" directive

but with latest master looks like it's gone

fixed by 4333a68ca0d23eba398a1fc68674988d05ac11dc
